### PR TITLE
Show check of which item is voted

### PR DESCRIPTION
**Pull request by bongsok**
Showing check of which item is voted of yes or no.
Disable buttons after voting.
Now double click should be needed to vote.

### CODE REVIEW

**Comment by bongsok** [comment]
We should consider how to update all counts after voting. This patch doesn't have updating counts of yes, no and total.

**Comment by JungoKim** [comment]
+1

File: client/src/app/components/new-asks.jsx [hunk 1]
```diff
@@ -19,6 +19,10 @@ var NewAsks = React.createClass({
     console.log('window.newAsksState is ', window.newAsksState);
     var query = {};
     query.date = new Date().getTime();
+    query.askerId = '';
+    if (document.user !== undefined) {
+      query.askerId = document.user.id || '';
```
Comment (luckypapa) [inline client/src/app/components/new-asks.jsx:24]: Is need duplication check?

File: client/src/app/components/new-asks.jsx [hunk 2]
```diff
@@ -60,6 +64,10 @@ var NewAsks = React.createClass({
     if (window.newAsksState === undefined || window.newAsksState === "UpdateNeeded") {
       var query = {};
       query.date = new Date().getTime();
+      query.askerId = '';
+      if (document.user !== undefined) {
+        query.askerId = document.user.id || '';
```
Comment (luckypapa) [inline client/src/app/components/new-asks.jsx:69]: Is need duplication check?

File: client/src/app/components/new-asks.jsx [hunk 3]
```diff
@@ -19,6 +19,10 @@ var NewAsks = React.createClass({
     console.log('window.newAsksState is ', window.newAsksState);
     var query = {};
     query.date = new Date().getTime();
+    query.askerId = '';
+    if (document.user !== undefined) {
+      query.askerId = document.user.id || '';
+    }
```
Comment (JungoKim) [inline client/src/app/components/new-asks.jsx:25]: no need

File: client/src/app/components/new-asks.jsx [hunk 4]
```diff
@@ -60,6 +64,10 @@ var NewAsks = React.createClass({
     if (window.newAsksState === undefined || window.newAsksState === "UpdateNeeded") {
       var query = {};
       query.date = new Date().getTime();
+      query.askerId = '';
+      if (document.user !== undefined) {
+        query.askerId = document.user.id || '';
+      }
```
Comment (JungoKim) [inline client/src/app/components/new-asks.jsx:70]: no need

File: client/src/app/components/new-asks.jsx [hunk 5]
```diff
@@ -85,6 +93,10 @@ var NewAsks = React.createClass({
     var query = {};
     var now = new Date().getTime();
     query.date = dateTime ? dateTime : now;
+    query.askerId = '';
+    if (document.user !== undefined) {
+      query.askerId = document.user.id;
+    }
```
Comment (JungoKim) [inline client/src/app/components/new-asks.jsx:99]: no need

File: client/src/app/components/vote-button.jsx [hunk 6]
```diff
@@ -134,52 +152,59 @@ var VoteButton = React.createClass({
...
-        console.log('!!!!!data: ' + JSON.stringify(data));
-        this.setState({valid: false});
-        this.refs.yesResult.show(data.yesCount);
-        this.refs.noResult.show(data.noCount);
-      }.bind(this),
-      error: function (xhr, status, err) {
-        console.error(url, status, err.toString());
-      }.bind(this)
-    });
+    if (this.lastTouchEvent && (e.nativeEvent.timeStamp - this.lastTouchEvent) < 250) {
```
Comment (JungoKim) [inline client/src/app/components/vote-button.jsx:155]: make a fuction and call function

File: client/src/app/components/vote-button.jsx [hunk 7]
```diff
@@ -134,52 +152,59 @@ var VoteButton = React.createClass({
...
-      success: function (data) {
-        this.setState({valid: false});
-        this.refs.yesResult.show(data.yesCount);
-        this.refs.noResult.show(data.noCount);
-      }.bind(this),
-      error: function (xhr, status, err) {
-        console.error(url, status, err.toString());
-      }.bind(this)
-    });
+    console.log('!!!!!!!_handleNoButton');
```
Comment (JungoKim) [inline client/src/app/components/vote-button.jsx:182]: ditto

**Pull request merged**